src/components/CurriculumComponents/CurriculumHeader/CurriculumHeader.test.tsx

```diff
@@ -77,16 +77,6 @@ describe("Component - Curriculum Header", () => {
   });
 
   test("user can see the tabular navigation", async () => {
-    const { findByTestId } = renderComponent();
```

Comment:
Should we also test to see we can see the downloads tab?

Reply:
We do test for that see line 91

```
expect(links[2]).toHaveTextContent("Download");
```